### ISSUE TITLE
Bug: This value is not valid.

### ISSUE THREAD

**Issue by 4n70w4** [ludofleury/satisfy#101]
I can't save the configuration, but I haven't even changed this value!

![image](https://user-images.githubusercontent.com/38257723/58875207-95259380-86d3-11e9-9c63-00cce21de390.png)

`git log`

```
commit 707803be30e1bdb5aa36dcb3fc91b13cccde9e37
Author: Ram<C5><AB>nas Dronga <github@ramuno.lt>
Date:   Sat May 25 17:54:56 2019 +0300
```

**Comment by ramunasd** [comment]
Repository name is required field nad without it json validation fails.

**Comment by 4n70w4** [comment]
@ramunasd why issue closed? The problem is not solved.
Repository name is filled. Where did you see that it is not filled?

**Comment by 4n70w4** [comment]
@rabauss your product contains a critical bug that does not allow editing settings.
Explain to me why issue is closed, but nothing is fixed?

**Comment by 4n70w4** [comment]
Post form. 
```
configuration[name]: Composer repository
configuration[description]: Sooqa
configuration[homepage]: http://rottvoy.ebal
configuration[requireDependencyFilter]: 1
configuration[minimumStability]: dev
configuration[includeFilename]: 
configuration[outputDir]: web
configuration[outputHtml]: 1
configuration[config]: 
configuration[twigTemplate]: 
configuration[notifyBatch]: 
configuration[prettyPrint]: 1
configuration[_token]: j7jkv3ueQq7zrKcj1QtrpnxEyeOKQXYO8z4eWGbZ3_M
```

Error: `This value is not valid.`
But `name` attribute is not empty!

**Comment by ramunasd** [comment]
This field has regexp constraint, to make satis.json compatible with upcomming changes in composer itself. Basically You could enter any string, but then satis would throw some unwanted deprecation warnings like this:
`Deprecation warning: Your package name Composer repository is invalid, it should have a vendor name, a forward slash, and a package name. The vendor and package name can be words separated by -, . or _. The complete name should match "[a-z0-9]([_.-]?[a-z0-9]+)*/[a-z0-9]([_.-]?[a-z0-9]+)*". Make sure you fix this as Composer 2.0 will error.`

So it is not a bug.
